bigbluebutton-html5/imports/ui/components/about/container.jsx

```diff
@@ -16,6 +16,8 @@ const getClientBuildInfo = () => (
   {
     clientBuild: Meteor.settings.public.app.html5ClientBuild,
     copyright: Meteor.settings.public.app.copyright,
+    bbbServerVersion: Meteor.settings.public.app.bbbServerVersion,
+    displayBbbServerVersion: Meteor.settings.public.app.displayBbbServerVersion,
```

Comment:
Does it make sense to refactor this slightly? `const { clientBuild, copyright, bbbServerVersion, displayBbbServerVersion } = Meteor.settings.public.app;`